BufferList.js

```diff
@@ -257,6 +261,41 @@ BufferList.prototype._appendBuffer = function appendBuffer (buf) {
   this.length += buf.length
 }
 
+BufferList.prototype.prepend = function append (buf) {
```

Comment:
maybe you should dedupe all this code with `append`?

```js
BufferList.prototype.append = function append (buf) {
  return attach(this, buf, 'append')
}

BufferList.prototype.prepend = function prepend (buf) {
  return attach(this, buf, 'prepend')
}

function attach (bl, buf, attachType) {
   ...
   bl[attachType](buf)
   ...
}
```

Reply:
Actually good point. I should not create PRs on Friday XD. Will update this very soon.

Reply:
I've refactored as you asked. :)